src/cf-typography/usage.md

```diff
@@ -316,61 +265,89 @@ attached to `.meta-header_left` and `.category-slug` is attached to `.meta-heade
 
 #### Default meta header
 
-<div class="meta-header">
-    <span class="meta-header_right date">
-        Nov 4, 2013
-    </span>
-    <a href="#" class="meta-header_left category-slug">
-        <span class="cf-icon cf-icon-credit-card"></span>
-        Consumer finance
-    </a>
+<div class="m-meta-header">
+    <div class="m-meta-header_right">
+        <span class="date">
+            Nov 4, 2013
+        </span>
+    </div>
+    <div class="m-meta-header_left">
+        <a href="#" class="m-meta-header_left category-slug">
+            <span class="cf-icon cf-icon-credit-card"></span>
+            Consumer finance
+        </a>
+    </div>
```

Comment:
Because there's typically multiple categories, and it's only by chance this worked out beforehand. By separating the layout concerns from content elements, we can guarantee the layout will always work.